app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugWithDosagesEvents.kt

```diff
@@ -0,0 +1,7 @@
+package org.simple.clinic.drugs.selectionv2
+
+import org.simple.clinic.widgets.UiEvent
+
+data class PrescribedDrugsWithDosagesSheetCreated(val drugName: String) : UiEvent
+
+object ScreenDestroyed: UiEvent
```

Comment:
We already have a `ScreenDestroyed` object (org.simple.clinic.widgets). Can you use that?

If you don't use that event, then your controller stream will never get disposed. This is a hidden behavior, but unfortunately `ReplayUntilScreenIsDestroyed` depends on the other event to dispose itself. We should fix this in the future.